test/e2e/internal/notation/registry.go

```diff
@@ -19,9 +19,16 @@ import (
 const ArtifactTypeNotation = "application/vnd.cncf.notary.signature"
 
 type Registry struct {
-	Host     string
+	// Host is the registry host.
+	Host string
+	// Username is the username to access the registry.
 	Username string
+	// Password is the password to access the registry.
 	Password string
+	// DomainHost is an registry host for testing --plain-http flag.
+	// if the host is localhost, notation make all connection as plain http.
+	// if the host is not localhost, notation make all connection as https.
```

Comment:
Is this statement true? even if host is not localhost notation can interact with registry using http.